tests/www_rbac/test_views.py

```diff
@@ -1464,12 +1464,9 @@ def test_trigger_dag_button_normal_exist(self):
 
     def test_trigger_dag_button(self):
 
-        test_dag_id = "example_bash_operator"
+        test_dag_id = "example_python_operator"
 
         DR = models.DagRun
-        self.session.query(DR).delete()
-        self.session.commit()
-
         self.client.get('trigger?dag_id={}'.format(test_dag_id))
```

Comment:
Hi @feng-tao , may be good to change this line to
```python
self.client.get('/trigger?dag_id={}'.format(test_dag_id))
```
(add the slash), even though seems it's not affecting that much.

Reply:
thanks. this pr is for testing only. I would like to see why the test fails particularly for MySQL ORM.